test/github-issues/9229/issue-9229.ts

```diff
@@ -0,0 +1,88 @@
+import sinon from "sinon"
+import {
+    DataSourceOptions,
+    ConnectionOptionsReader,
+    DataSource,
+    DatabaseType,
+} from "../../../src"
+import {
+    setupTestingConnections,
+    createTestingConnections,
+    closeTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { CommandUtils } from "../../../src/commands/CommandUtils"
+import { MigrationGenerateCommand } from "../../../src/commands/MigrationGenerateCommand"
+import { TestEntity } from "./entity/test.entity"
+import { User } from "./entity/user"
+import path from "path"
+
+describe("github issues > #9229 for some reason, TypeORM generates a second migration file after running all migrations", () => {
```

Comment:
you need to simplify test case... is there any reason why it's doesn't follow the same pattern other test cases do? I would like to keep everything consistent to make it easier to maintain.

Reply:
Yeah I simplified now. Thank you for your guidance.